stores/AISShips/AISShipsReducers.ts

```diff
@@ -0,0 +1,49 @@
+import BaseReducer from "@/utils/BaseReducer";
+import AISShipsActions from "./AISShipsActions";
+import { AISShipsState } from "./AISShipsTypes";
+import { AnyAction } from "redux";
+
+export default class AISShipsReducer extends BaseReducer {
+    initialState: AISShipsState = {
+        data: {
+            ships: [
+                {
+                    id: 0,
+                    latitude: 42.1232,
+                    longitude: -123.3045,
```

Comment:
This also may need updating (it lands somewhere in Oregon), but we can worry about this later. When you start doing the UI work, then it will be necessary to start thinking where we visualize the boats